cmd/entrypoint/runner.go

```diff
@@ -21,12 +23,38 @@ func (*realRunner) Run(args ...string) error {
 	}
 	name, args := args[0], args[1:]
 
+	// Receive system signals on "signals"
+	signals := make(chan os.Signal, 1)
+	defer close(signals)
+	signal.Notify(signals, syscall.SIGINT, syscall.SIGTERM)
+	defer signal.Reset()
+
 	cmd := exec.Command(name, args...)
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
+	// dedicated PID group used to forward signals to
+	// main process and all children
+	cmd.SysProcAttr = &syscall.SysProcAttr{Setpgid: true}
+
+	// Goroutine for signals forwarding
+	go func() {
+		for s := range signals {
+			// Forward signal to main process and all children
+			syscall.Kill(-cmd.Process.Pid, s.(syscall.Signal))
+		}
+	}()
+
+	// Start defined command
+	err := cmd.Start()
```

Comment:
Nit: `if err := ...; err != nil` is a bit more idiomatic and consistent, here and below.

Reply:
I have updated this and added tests as well.